g/github.com_jinzhu_gorm/github.com_jinzhu_gorm_1.9.1_UBI_8.5.sh

```diff
@@ -0,0 +1,58 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : github.com/jinzhu/gorm
+# Version        : v1.9.1
+# Source repo    : https://github.com/jinzhu/gorm
+# Tested on      : UBI 8.4
+# Language      : go
+# Travis-Check  : True
+# Script License : Apache License, Version 2 or later
+# Maintainer     : Vaibhav Bhadade <vaibhav.bhadade@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash -e
+
+PACKAGE_NAME=github.com/jinzhu/gorm
+PACKAGE_PATH=https://github.com/jinzhu/gorm
+PACKAGE_VERSION=${1:-v1.9.1}
+GO_VERSION="go1.17.5"
```

Comment:
I don't think this is needed and may cause confusion as we installing go using yum and its not at version 1.17.5

Reply:
> I don't think this is needed and may cause confusion as we installing go using yum and its not at version 1.17.5

go version line removed , not needed